README.md

```diff
@@ -50,6 +61,7 @@ Many Windows developers have found that setting up an Ubuntu virtual machine is
 1. `cd code-dot-org/aws`
 1. `bundle install`
 1. `cd ..`
+1. <code>sudo chown \`whoami\` ../.npm</code>
```

Comment:
is this line necessary for a regular npm installation, or is it just a failsafe in case something messed up with permissions? If it's necessary, I'd prefer this line reads `sudo chown $(whoami) $HOME/.npm` in case the users aren't installing somewhere other than `$HOME/code-dot-org`. If not necessary, maybe leave this line out to avoid confusion